hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -214,6 +216,11 @@ public OzoneManagerProtocolClientSideTranslatorPB(OzoneConfiguration conf,
     this.clientID = clientId;
   }
 
+  public OzoneManagerProtocolClientSideTranslatorPB(OzoneConfiguration conf,
```

Comment:
This can be removed, no one is calling this.

Reply:
@bharatviswa504 It turns out there is one caller here: https://github.com/apache/hadoop/blob/d69a1a0aa49614c084fa4b9546aceeeee65aebe4/hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/ReconControllerModule.java#L102

But we can easily change it to use the new constructor. Shall we do that?

Reply:
Yes if possible.